app/src/main/java/com/keylesspalace/tusky/view/TuskySwipeRefreshLayout.kt

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2024 Tusky contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+package com.keylesspalace.tusky.view
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+import com.google.android.material.color.MaterialColors
+
+/**
+ * SwipeRefreshLayout does not allow theming of the color scheme,
+ * so we use this class to still have a single point to change its colors.
+ */
+class TuskySwipeRefreshLayout @JvmOverloads constructor(
```

Comment:
Should(n't) this be a style file? And all swipe to refresh have that as "style"?

Reply:
You mean adding the missing styles here and then styling every SwipeRefreshLayout? Since the styling would be all the same and would need quite some additional lines of code I'd rather not.

Reply:
I was thinking how all the buttons for example are style with some definition in styles.xml and the a "style=" on every usage.

But I couldn't find now how that might be done.